src/blocks/time.rs

```diff
@@ -107,8 +111,16 @@ pub async fn run(config: &Config, api: &CommonApi) -> Result<()> {
 
         api.set_widget(widget)?;
 
+        let tick = async {
```

Comment:
Why make this an async block instead of just:

```rust
        let phase = now.second() as u64 % config.interval.seconds();
        if phase != 0 {
            timer.reset_after(Duration::from_secs(config.interval.seconds() - phase));
        }
```

Reply:
Yeah, that makes sense.